src/apps/material/helper.ts

```diff
@@ -157,7 +157,15 @@ export const getManifestationOriginalTitle = (manifestation: Manifestation) => {
 export const getManifestationContributors = (manifestation: Manifestation) => {
   return (
     manifestation.contributors
-      .map((contributor) => contributor.display)
+      .map((contributor) => {
+        if (contributor.roles.length > 0) {
+          const roleNames = contributor.roles
```

Comment:
These TS errors are a bit alarming.
It seesm like you did not run the FBI codegen cmd after updating the fragment?

Reply:
Hi @spaceo
The schema was updated in this commit: https://github.com/danskernesdigitalebibliotek/dpl-react/pull/1146/commits/0b2405e4d24ff485b1689ee2f1195555dc68b64c#diff-d7e14ab57cd19ae792d595c71bf36d4157c7fea99e0a7ea96e1003494b0baf6e 🙂